Code/autopkglib/URLDownloader.py

```diff
@@ -125,7 +126,8 @@ def main(self):
         # Download URL.
         url_handle = None
         try:
-            request = urllib2.Request(url=self.env["url"])
+            url = urllib.quote(self.env["url"], 'http://')
```

Comment:
for some reason, I can't specify just the colon, i.e.:

url = urllib.quote(self.env["url"], ':')

I does not work either.